app/graphql/types/compute_resource.rb

```diff
@@ -13,5 +13,9 @@ class ComputeResource < BaseObject
     has_many :compute_attributes, Types::ComputeAttribute
     has_many :hosts, Types::Host
     has_many :networks, Types::Networks::Union, resolver: Resolvers::ComputeResource::Networks
+
+    def provider_friendly_name
+      object.provider_friendly_name.tr(' ', '_')
```

Comment:
I didn't figure out how to nicely write a test for this.

Reply:
I think the existing query test is enough, wdyt?

https://github.com/theforeman/foreman/blob/f49802123759130ab64b54dc35ac76120bc64b0d/test/graphql/queries/compute_resource_query_test.rb#L81